<launch>
    <include file="$(find turtlebot_bringup)/launch/minimal.launch"/>
    <include file="$(find turtlebot_bringup)/launch/3dsensor.launch"/>

    <node pkg="c2" name="c2_main" type="c2_main.py" output="screen"/>
    <node pkg="uvc_camera" type="uvc_camera_node" name="uvc_camera" output="screen">
        <param name="device" type="string" value="/dev/video1" />
        <remap from="/image_raw" to="usb_cam/image_raw"/>
    <!-- <param name="width" type="int" value="320" />
    <param name="height" type="int" value="240" />
    <param name="fps" type="int" value="30" />
    <param name="frame" type="string" value="wide_stereo" />

    <param name="auto_focus" type="bool" value="False" />
    <param name="focus_absolute" type="int" value="0" />

    <param name="device" type="string" value="/dev/video0" />
    <param name="camera_info_url" type="string" value="file://$(find uvc_camera)/example.yaml" /> -->
    </node>

    <!--node pkg="smach_viewer" type="smach_viewer.py" name="smach_viewer" /-->
    <!--node pkg="rviz" type="rviz" name="rviz" /-->
</launch>
